assets/aml-benchmark/components/inference-postprocessor/spec.yaml

```diff
@@ -0,0 +1,94 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+name: inference_postprocessor
+display_name: Inference Postprocessor
+description: Inference Postprocessor
+version: 0.0.19
+is_deterministic: true
+
+inputs:
+  prediction_dataset:
+    type: uri_file
+    description: >-
+      A file that contains predicted values
+    optional: False
+  prediction_column_name:
```

Comment:
nit: If there is one column in the prediction, why don't we just use that and make `prediction_column_name` optional? Same applies for `ground_truth_column_name`. This reduces the parameters that we have to configure.

Reply:
@arun-rajora I tried this, but template was not able to pick 0 as column name to apply logic on predicted value in jinja template. So, I added the prediction_column_name as required.